History/src/helpers/callsToAuth.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const env = require('../loadEnvironment.js');
```

Comment:
I think as standardization with the rest, the `.js` extension should be removed.